query/sources/common/src/main/java/org/codice/ddf/admin/sources/utils/RequestUtils.java

```diff
@@ -225,4 +216,17 @@ private WebClient generateClient(String url, CredentialsField creds,
         }
         return client;
     }
+
+    private ResponseField responseFieldFromResponse(Response response, UrlField requestUrl) {
+        // Check if we received a content type, if not leave it blank since we don't know what it is
+        String contentType = response.getMediaType() == null ?
```

Comment:
There may be a difference between an empty string and null in graphql. It might be safer to leave contentype as null.